tests/test_inverses.py

```diff
@@ -178,17 +178,3 @@ def circuit():
             return qml.expval(qml.PauliZ(0))
 
         assert np.allclose(circuit(), expected_output, atol=tol, rtol=0)
-
```

Comment:
This test case coincides with the following PL test:
https://github.com/XanaduAI/pennylane/blob/e180c7ed5a005277a986222879ed9b569b83bb7d/tests/test_device.py#L238